src/main/java/com/example/userlist/data/repository/UsersRepositoryImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.example.userlist.data.repository;
+
+import com.example.userlist.data.dao.UserDao;
+import com.example.userlist.domain.repository.UsersRepository;
+import com.example.userlist.domain.entity.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class UsersRepositoryImpl implements UsersRepository {
```

Comment:
You already have UserDao, why do you need a repository too?

Reply:
I asked you yesterday about "do i need a repository abstraction, is it necessary" and you said that I can do it.

Reply:
It is abstraction under abstraction. What is the point to have such nested absraction? Or just leave DAO or Repository.